homeassistant/components/speedtestdotnet/sensor.py

```diff
@@ -101,4 +100,4 @@ async def async_added_to_hass(self) -> None:
         await super().async_added_to_hass()
         state = await self.async_get_last_state()
         if state:
-            self._attr_native_value = state.state
+            self._state = state.state
```

Comment:
Maybe add a test covering this, look into the `mock_restore_cache` helper in `tests/common.py` 👍